hermes-api/src/main/java/pl/allegro/tech/hermes/api/Stats.java

```diff
@@ -0,0 +1,46 @@
+package pl.allegro.tech.hermes.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+public class Stats {
+    private final TopicStats topicStats;
```

Comment:
What do you think about using @Nonnull property? Do you use it? Maybe you (as a team) should consider using it

Reply:
There is no such convention right now so I'd rather not change that in this PR. However this something to consider in the future